src/app/views/query-response/QueryResponse.tsx

```diff
@@ -22,23 +22,39 @@ import { getPivotItems, onPivotItemClick } from './pivot-items/pivot-items';
 import './query-response.scss';
 import { IRootState } from '../../../types/root';
 
+
 const QueryResponse = (props: IQueryResponseProps) => {
   const dispatch = useDispatch();
 
   const [showShareQueryDialog, setShareQuaryDialogStatus] = useState(true);
   const [showModal, setShowModal] = useState(false);
   const [query, setQuery] = useState('');
   const [responseHeight, setResponseHeight] = useState('610px');
-
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
   const { dimensions, sampleQuery } = useSelector((state: IRootState) => state);
 
+  let isTabletSize : boolean = (windowWidth <= 1320) ? true : false;
+
   const {
     intl: { messages },
   }: any = props;
 
   useEffect(() => {
     setResponseHeight(convertVhToPx(dimensions.response.height, 50));
-  }, [dimensions]);
+    window.addEventListener("resize", handleWindowResize, false);
+    return () => window.removeEventListener('resize', handleWindowResize);
+  }, [dimensions, windowWidth]);
+
+  const handleWindowResize = () => {
+    const currentWindowWidth = window.innerWidth;
+    if(currentWindowWidth <= 1320){
+      isTabletSize = true;
+    }
+    else{
+      isTabletSize = false;
+    }
```

Comment:
*suggestion*: This can also be summarised to
```suggestion
let isTabletSize = false;
    if(currentWindowWidth <= 1320){
      isTabletSize = true;
    }
```

Reply:
Thank you. Let me make the change